### ISSUE TITLE
Handeling time expensive tasks with Celery

### ISSUE THREAD

**Issue by timatooth** [jdriscoll/django-photologue#100]
Hi there,

I'm thinking of extending photologue so that time intensive tasks such as uploading zips, image resampleing or applying watermarks gets done in a Celery worker backed by a message broker such as rabbitMQ. Would you be interested in merging something like that into the project if I tried to keep it as an optional thing to configure?

**Comment by bijanvakili** [comment]
> I think that it's something that should stay out of the core Photologue code, as for most users, it's not necessary - and, if performance is an issue and suggests the use of asynchronous Qs, Celery is not the only option anyway. Personally, I've tested both Celery and django-rq, and I far prefer django-rq! But that's just a personal opinion. My point is, there is more than one asynchronous Q solution out there, and it's often not needed anyway.

I agree that it should probably stay out of the code.  It might be simpler to add something like this to the `example_project` directory.

I can also confirm that Celery is not the only option.  I was able to get asynchronous batch imports into Photologue using [procrastinate with Django](https://procrastinate.readthedocs.io/en/stable/howto/django/configuration.html) in my own Django application that includes photologue as a dependency.
